src/utils/questionnaireBuilderUtils.ts

```diff
@@ -1,245 +1,331 @@
-  return {
-    title: "",
-    description: "",
-    questions: [],
-    passScore: 0,
-    tags: [],
-  };
+export const isFormValid = (questionnaire: Questionnaire): boolean => {
+  const { isValid } = validateQuestionnaire(questionnaire);
+  return isValid;
 };
 
 /**
- * Function to check if form is valid for submission
+ * Gets appropriate tooltip message for a questionnaire's validation state
+ * Using only existing localization strings
  * 
- * @param questionnaire Current questionnaire state
- * @returns Boolean indicating if form is valid
+ * @param questionnaire - The questionnaire to get tooltip message for
+ * @param strings - The localization strings object
+ * @returns Localized tooltip message based on validation result
  */
-export const isFormValid = (questionnaire: Questionnaire): boolean => {
-  // Check required fields
-  if (!questionnaire.title || !questionnaire.description) {
-    return false;
+export const getValidationTooltipMessage = (
+  questionnaire: Questionnaire, 
+  strings: any
+): string => {
+  const { errorType } = validateQuestionnaire(questionnaire);
+  
+  if (!errorType) {
+    return ""; // No error
   }
-  if (!questionnaire.questions || questionnaire.questions.length === 0) {
-    return false;
+  
+  // Map error types to existing localized strings
+  switch(errorType) {
+    case "bothEmpty":
+      return strings.newQuestionnaireBuilder.tooltipBothEmpty;
+    case "emptyTitle":
+      return strings.newQuestionnaireBuilder.tooltipEmptyTitle;
+    case "emptyDescription":
+      return strings.newQuestionnaireBuilder.tooltipEmptyDescription;
+    case "noQuestions":
+      return strings.error.questionnaireSaveFailed + ", " + "At least one question is required";
+    case "noCorrectAnswers":
+      return strings.error.questionnaireSaveFailed + ", " + "At least one answer must be marked as correct";
+    case "noPassScore":
+      return strings.error.questionnaireSaveFailed + ", " + "Please set a required score using the slider";
+    default:
+      return strings.error.generic;
   }
-  // Check if any question has at least one correct answer
-  const hasAnyCorrectAnswers = questionnaire.questions.some(question => 
-    question.answerOptions && question.answerOptions.some(option => option.isCorrect)
-  );
+};
 
-  if (!hasAnyCorrectAnswers) {
-    return false;
+/**
+ * Legacy function for backward compatibility
+ * @deprecated Use getValidationTooltipMessage instead
+ */
+export const getTooltipMessage = (questionnaire: Questionnaire): string => {
+  if (!questionnaire.title && !questionnaire.description) {
+    return "Title and description are required";
   }
-
-  if (questionnaire.passScore === undefined || questionnaire.passScore === null) {
-    return false;
+  if (!questionnaire.title) {
+    return "Title is required";
+  }
+  if (!questionnaire.description) {
+    return "Description is required";
   }
-  return true;
+  return "";
 };
```

Comment:
If this is no longer used it should be removed, as we are still in staging data we can just remove/ update  any of the questionanires that break without this.